app/services/clock.js

```diff
@@ -1 +1,5 @@
-export { default } from 'ember-clock/services/clock';
\ No newline at end of file
+import ENV from '../config/environment';
+import Service from 'ember-clock/services/clock'
+export default Service.extend({
+  disabled: ENV['ember-clock'] && ENV['ember-clock'].disabled
```

Comment:
@rbezemer When I run the tests I get this error
```
Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties (ember/avoid-leaking-state-in-ember-objects)
```

Can you set the disabled flag in a constant like this:

```js
const disabled = ENV['ember-clock'] && ENV['ember-clock'].disabled

export default Service.extend({
  disabled
})
```